dev_requirements.txt

```diff
@@ -4,6 +4,7 @@ qiskit-aer~=0.8.2
 qiskit-ibmq-provider~=0.16.0
 pyquil~=2.28.0
 amazon-braket-sdk~=1.5.10
+pennylane-qiskit~=0.16.0
```

Comment:
Thanks, I've posted an issue to share with the team regarding the chained dependency on all of qiskit: https://github.com/PennyLaneAI/pennylane-qiskit/issues/149

Reply:
Thanks!